engine/vm.go

```diff
@@ -302,6 +303,19 @@ func (vm *VM) SetUserOutput(s *Stream) {
 	vm.output = s
 }
 
+// ResetEnv is used to reset all global variable
+func (vm *VM) ResetEnv() {
+	atomic.StoreInt64(&varCounter, 0)
```

Comment:
@ccamel Yes, you're right, I think atomicity here is not necessary since by implementing this kind of workaround, concurrent interpreter is no longer possible and must be avoided. By the way, I don't think it's completely possible due to the usage of global variable for env in the original library (and not only for `varCounter`). So I suggest this if it's ok fo you ? 

```suggestion
	varCounter = 0
```

Reply:
Yes, that's fine.